versioned_docs/version-8.7/guides/getting-started-java-spring.md

```diff
@@ -96,7 +96,7 @@ To deploy your process, take the following steps:
 4. Change the **Cluster endpoint** to `http://localhost:26500`, with no authentication.
 5. Click **Deploy**.
 
-When you open Operate at http://localhost:8080/operate/, you should now note the process deployed to your local Self-Managed setup.
+When you open Operate at http://localhost:8080/operate/ (or http://localhost:8081 using Docker Compose), you should now note the process deployed to your local Self-Managed setup.
```

Comment:
I want to keep the focus on C8Run as a Java application for now, but the difference in ports highlights an issue with the experience... 

Maybe instead, we could qualify this as "Camunda 8 Run with Docker Compose". WDYT?

Reply:
I've updated it to be specific - how is this version?

Reply:
Fab!